gtk-3.22/gtk-widgets.css

```diff
@@ -1748,16 +1749,16 @@ scrollbar.horizontal slider {
     min-height: 3px;
 }
 
-scrollbar.horizontal:hover slider {
-    min-height: 8px;
-}
-
 scrollbar.vertical slider {
     min-height: 30px;
     min-width: 3px;
 }
 
-scrollbar.vertical:hover slider {
+scrollbar.overlay-indicator.horizontal:hover slider {
```

Comment:
This means non-overlay scrollbars are always small and can never expand. This would really break click targets here.

Reply:
hm... i thought, expanded non-overlay scrollbars was due to simpler code (selector). When such scrollbar expands, it causes content to resize too. I took example from Adwaita, where expanding was feature of overlay scrollbars, while non-overlay was at full size. However, you then wrote, that they need to be more narrow.

So, should static scrollbars indeed be expanded?
(i also asked for that about to make its width between default and hovered width - i.e. 5 or 6px)

Reply:
@danrabbit When non-scrollbar expands, its junction produces artifact, described above. But it doesn't happen if scrollbar doesn't change size.

When junction resizes, new border is drawed, bug old area os not repainted with normal background color. When slider is dragged, this somehow causes complete junction redraw with background, but when dragging is completed and scrollbar is unhovered, this produces second sort of artifact.

Reply:
commited